common/djangoapps/entitlements/models.py

```diff
@@ -24,3 +24,25 @@ class CourseEntitlement(TimeStampedModel):
         help_text='The current Course enrollment for this entitlement. If NULL the Learner has not enrolled.'
     )
     order_number = models.CharField(max_length=128, null=True)
+
+    @classmethod
+    def get_active_user_course_entitlements(cls, user, course_uuid):
```

Comment:
Nit: The method name is plural, but the logic only seems like it would ever return a single entitlement.

Reply:
It will return multiple entitlements.